File: src/FumbleFunds.Api/Models/Entity/Match.cs
using System.ComponentModel.DataAnnotations;
using fumble_funds.Models.Enums;
public class Match
    {
        public int Id { get; set; }

        [Required]
        public string HomeTeam  { get; set; } = string.Empty;

        [Required]
        public string AwayTeam  { get; set; } = string.Empty;

        public DateTime StartTime { get; set; }

        // scores only set once Finished
        public int? HomeScore { get; set; }
        public int? AwayScore { get; set; }

        public MatchStatus Status { get; set; } = MatchStatus.Scheduled;

        // nav
        public ICollection<Bet> Bets { get; set; } = new List<Bet>();
    }

File: frontend/src/models/Match.ts
import {MatchStatus} from "./MatchStatus.ts";

export interface Match {
    id: number;
    homeTeam: string;
    awayTeam: string;
    startTime: Date;
    homeScore: number;
    awayScore: number;
    status: MatchStatus;
    bets: string[];
}